lib/qa/authorities/getty.rb

```diff
@@ -50,8 +50,7 @@ def find_url id
     end
 
     def request_options
-      # Don't pass a request header. See http://answers.semanticweb.com/questions/31906/getty-sparql-gives-a-404-if-you-pass-accept-applicationjson
-      { }
+      { accept: 'sparql-results+json'}
```

Comment:
I assume there's something later in the chain that transforms this to `application/sparql-results+json`?